examples/TodoApp.spec.js

```diff
@@ -1,32 +1,49 @@
 /**
  * This is the example app used in the documentation.
  * If you want to run it, you will need to build the final bundle with
- * yarn build. Then you can run this with `yarn test examples`
+ * pnpm build. Then you can run this with `pnpm test examples`
```

Comment:
I think the test file should be in TypeScript (and all references in docs should be updated accordingly)